{
  final Exception thrown=new Exception();
  hazelcastLogger.warning("message",thrown);
  verify(mockLogger,times(1)).warn("message",thrown);
}
